package.json

```diff
@@ -43,7 +44,7 @@
 		"dot-prop": "^5.1.0",
 		"esm": "^3.2.18",
 		"execa": "^2.0.4",
-		"got": "^9.3.2",
+		"got": "^11.8.2",
```

Comment:
These changes should not appear in this PR. Could you rebase the `main` branch?

`git rebase -i main`. Then some conflicts need to be resolved when rebasing.

Reply:
If I remove `c0f6e87`, I think I also have to remove the 4 commits below to pass the tests (ava && tsd).

May I ask it is alright to remove these 5 commits?

Reply:
```shell
git checkout main
git pull upstream main
git checkout Add-types
git rebase -i main
git push origin Add-types -f
```

Reply:
Oh, I got it, thanks for guiding me :)